include/pgduckdb/pg/relations.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "pgduckdb/pg/declarations.hpp"
+
+namespace pgduckdb {
+
+// FIXME - this is a macro in Postgres, would be great to inline it to avoid the overhead of a function call
```

Comment:
Do we actually care about this? We don't call this for every row right?

Reply:
Yeah true, wanted to flag it though. I've removed the comment for now.